### ISSUE TITLE
e2e flake: TestPauseControl

### ISSUE THREAD

**Issue by hongchaodeng** [coreos/etcd-operator#532]
```
07:00:37 --- FAIL: TestPauseControl (105.24s)
07:00:37 	e2e_test.go:215: failed to resize to 3 members etcd cluster: still failing after 6 retries
```

**Comment by hongchaodeng** [comment]
From the logs, it successfully paused.
But when it transitioned to non-paused, it keeps failing to add new member:
```
fail to add new member (test-etcd-r2lgi-0003): context deadline exceeded" cluster-name=test-etcd-r2lgi pkg=cluster 
```

**Comment by hongchaodeng** [comment]
etcd pod logs when failing to add member (0003). One leader (0002) one follower (0001):

leader:
```
2016-12-28 07:12:17.205993 I | raft: 3a3f64bba6426e4e became leader at term 14
...
2016-12-28 07:12:17.293485 W | etcdserver: failed to reach the peerURL(http://test-etcd-sj6t9-0003:2380) of member ec5dad2586e5c6cc (Get http://test-etcd-sj6t9-0003:2380/version: dial tcp: lookup test-etcd-sj6t9-0003 on 10.39.240.10:53: server misbehaving)
2016-12-28 07:12:17.293651 W | etcdserver: cannot get the version of member ec5dad2586e5c6cc (Get http://test-etcd-sj6t9-0003:2380/version: dial tcp: lookup test-etcd-sj6t9-0003 on 10.39.240.10:53: server misbehaving)
2016-12-28 07:12:18.653679 W | membership: Reject add member request: the number of started member (2) will be less than the quorum number of the cluster (3)
2016-12-28 07:12:18.653886 W | etcdserver: not enough started members, rejecting member add {ID:fb3e1a5232f1ebda RaftAttributes:{PeerURLs:[http://test-etcd-sj6t9-0003:2380]} Attributes:{Name: ClientURLs:[]}}
...
```

follower:
```
2016-12-28 07:12:17.204071 I | raft: 452069f9773f271f became follower at term 14
...
```

**Comment by hongchaodeng** [comment]
Figured out the root cause:

The operator has actually added the member before, but the request failed:
```
time="2016-12-28T18:47:36Z" level=error msg="fail to add new member (test-etcd-bjpp0-0003): context deadline exceeded" cluster-name=test-etcd-bjpp0 pkg=cluster 
```

On etcd pod:
```
2016-12-28 18:47:41.202678 I | membership: added member b33cfb93faa53d3 [http://test-etcd-bjpp0-0003:2380] to cluster a4d5d4d81c0b0e06
```

When operator tried again, it gets error -- because when adding one more member, etcd thinks it will have 4 members.

**Comment by hongchaodeng** [comment]
closed via #559
